src/main/java/error/ErrorMessage.java

```diff
@@ -0,0 +1,19 @@
+package error;
+
+public enum ErrorMessage {
+    DUPLICATED_NAMES("중복된 이름이 존재합니다."),
+    INVALID_NAME_LENGTH("이름은 5글자 이하여야 합니다."),
+    INVALID_COUNT("시도 횟수는 1회 이상이여야 합니다."),
```

Comment:
나중에 이름은 7글자 이하가 되야한다던가, 시도 횟수가 5회 이상이어야 한다고 요구사항이 바뀌면 어떻게 해야할까요?

Reply:
```suggestion
    INVALID_NAME_LENGTH("이름은 " + Name.MAX_NAME_LENGTH + "글자 이하여야 합니다."),
    INVALID_COUNT("시도 횟수는 " + TryCount.MIN_COUNT + "회 이상이여야 합니다."),
```
이름의 길이와 시도 횟수의 검증을 확인하는 각 클래스 내부에 선언된 상수를 public으로 수정하였습니다.
요구 사항이 변경될 경우 해당 검증을 수행하는 클래스 내부의 상수 값을 변경해주면 유지 보수에 용이할 것 같습니다.

Reply:
좋은 시도군요! 더 나아가서 MAX_NAME_LENGTH나 MIN_COUNT를 ErrorMessage가 모르도록 String.format을 사용해보셔도 좋을 것 같습니다. 객체지향의 특징 중 하나는 캡슐화와 정보은닉이니까요 😉

Reply:
```java
INVALID_NAME_LENGTH("이름은 %d글자 이하여야 합니다.")
```
```java
throw new IllegalArgumentException(
                    String.format(ErrorMessage.INVALID_NAME_LENGTH.getValue(), MAX_NAME_LENGTH)
            );
```
String.format을 적용함으로써, public으로 선언해 두었던 상수를 private으로 변경하였습니다.